00-HTML-CSS-basics/HTML/Excersise1/helloWorld.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
We should try to use the following naming convention for files, this can cause some issues in the future. As example this file should be called:

`hello-world.html` instead of `helloWorld.html`